services/file_system/project/api/manage_s3.py

```diff
@@ -150,63 +249,83 @@ def get(self):
 
         if user_id == None or user_id == '' :
             responseObject['message'] = "No user_id found"
-            return jsonify(responseObject), 400
-
-        # Get Key_name
-        if 'key_name' not in request.values:
-            responseObject['message'] = "No key name given"
-            return jsonify(responseObject), 400
-        elif request.values.get('key_name') == '':
-            responseObject['message'] = "No key name given"
-            return jsonify(responseObject), 400
-        else:
-            key_name = request.values.get('key_name')
+            return make_response(jsonify(responseObject)), 400
 
-        # Get file_name
-        if 'file_name' not in request.values:
-            responseObject['message'] = "No file name to save as"
-            return jsonify(responseObject), 400
-        elif request.values.get('file_name') == '':
-            responseObject['message'] = "No file name to save as"
-            return jsonify(responseObject), 400
-        else:
-            file_name = request.values.get('file_name')
 
         # Get bucket_name
         if (request.values.get('bucket_name') == 'uploads'):
             bucket_name = os.getenv('S3_UPLOAD')
             file_type = "uploads"
+            download_url = "https://s3-us-west-2.amazonaws.com/capstone.upload/"
```

Comment:
Hardcoded, could be config values instead?

Reply:
agreed, try to abstract out any constant URLs

Reply:
Try to stay consistent with using ' or ".